README.md

```diff
@@ -41,3 +41,38 @@ If you want to deploy your own HomeApp which supports Google Login, make sure yo
 1. Click "Google".  
 1. Enter email address.  
 1. Click "Register".  
+
+
+# Using nginx to serve
+
+To serve home app with nginx, you can use build script to prepare stuff.
+
+```bash
+CELL_NAME=app-cc-home UNIT_FQDN=demo.personium.io bash ./build.sh 3> build.log
+```
+
+And then, you can use built stuff in `dst` folder to serve home app.
+
+## build variables
+
+You can specify below environment variables to configure builds.
+
+|env|description|
```

Comment:
Can you add example column and each value?

Reply:
I added examples. Please review that.